OracleHTTPServer/dockerfiles/12.2.1.4.0/Dockerfile

```diff
@@ -77,19 +71,23 @@ RUN yum install -y unzip libaio sysstat make psmisc gcc && \
 # Go to /u01 as user 'oracle' to proceed with OHS installation
 #--------------------------------------------------------
 USER oracle
-RUN cd /u01 && unzip -q /u01/$OHS_PKG && cd - && \
-    /u01/$OHS_BIN -silent -responseFile /u01/install.file -invPtrLoc /u01/oraInst.loc ORACLE_HOME=$ORACLE_HOME && \ 
-    rm /u01/$OHS_BIN  /u01/oraInst.loc /u01/install.file && \
-    mkdir /u01/oracle/container-scripts/ /u01/oracle/logs 
+WORKDIR /u01
+#RUN unzip -q /u01/${OHS_PKG} && cd - && \
+RUN unzip -q /u01/${OHS_PKG} && \
+    /u01/${OHS_BIN} -silent -responseFile /u01/install.file -invPtrLoc /u01/oraInst.loc ORACLE_HOME=${ORACLE_HOME} && \ 
+    rm /u01/${OHS_BIN}  /u01/oraInst.loc /u01/install.file && \
+    mkdir /u01/oracle/container-scripts/ /u01/oracle/logs && \
+    chown oracle:oracle -R /u01/oracle/logs 
 
 # Copy required files to build this image
 # ------------------------------------------------------
-COPY container-scripts/* /u01/oracle/container-scripts/
+COPY container-scripts/* /u01/oracle/
 
 #Expose all Ports
-EXPOSE $OHS_SSL_PORT $OHS_LISTEN_PORT $NM_PORT
+EXPOSE ${OHS_SSL_PORT} ${OHS_LISTEN_PORT} ${NM_PORT}
 
 # Final setup
+USER oracle
 WORKDIR /u01/oracle/container-scripts
```

Comment:
I think this will also need modification.

Reply:
resolved